### ISSUE TITLE
Frame index mismatch between GOPRO_Large and GOPRO_Large_all?

### ISSUE THREAD

**Issue by gmy001** [SeungjunNah/DeepDeblur_release#66]
Hello, thanks for sharing the great dataset. I am trying to generate my own blur frames using frames in GOPRO_Large_all following the process you described in #59 . However, I met an inconsistency issue of mismatched frame index.
For example, in the `GOPRO_Large/test/GOPR0384_11_05,` the first blur frame is `004001.png,` which if I understand correctly, will be an average of frame `GOPRO_Large_all/test/GOPR0384_11_05/044001.png~044011.png.` However, when I did the average, the result is not the same, there seems to be a few frames missing.
I also find that `GOPRO_Large_all/test/GOPR0384_11_05/044001.png` happens to be exactly the same as `GOPRO_Large/test/GOPR0384_11_05/sharp/004001.png.` Shouldn't the `sharp/004001.png` be the same as `GOPRO_Large_all/test/GOPR0384_11_05/044006.png,` since it's the middle of the first 11 frames?

This issue happens also in several other clips. Is there any wrong with my blur generation process?

**Comment by DachunKai** [comment]
Yes, I found the same problem. Have you found which frames are used to simulate `GOPRO_Large/test/GOPR0384_11_05/blur/004001.png.` in `GOPRO_Large_all`?
